src/core/nodeOps.ts

```diff
@@ -136,65 +134,52 @@ export const nodeOps: RendererOptions<TresObject, TresObject> = {
   },
   remove(node) {
     if (!node) return
+    const ctx = node.__tres
```

Comment:
**suggestion:** It would be nice if `instance.__tres` was typed.

Reply:
Check https://github.com/Tresjs/tres/pull/522/files/aaa4d322d3e049c1ebd8b1f0362bca464471067e#r1472657228